XYZDrivers/src/main/java/com/xyzdrivers/repositories/ClaimsRepo.java

```diff
@@ -1,49 +1,58 @@
 package com.xyzdrivers.repositories;
 
 import com.xyzdrivers.models.Claim;
+import java.lang.reflect.Field;
+import java.sql.Date;
+import java.lang.reflect.Field;
 import java.sql.SQLException;
-import java.time.LocalDate;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Calendar;
 import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import javax.enterprise.context.RequestScoped;
 
 /**
  * DAO for claims model
- * 
+ *
  * @author arthur
  */
 @RequestScoped
 public class ClaimsRepo extends Repo<Claim, Integer> {
-        
+
     @Override
     public Claim get(Integer id) throws RepositoryException {
         throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
     }
 
     /**
      * Retrieve all claims
-     * 
+     *
      * @return ArrayList<Claim> All claims
      * @throws com.xyzdrivers.repositories.RepositoryException
      */
     @Override
     public List<Claim> get() throws RepositoryException {
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
         List<Object[]> results;
         List<Claim> claims = new ArrayList<>();
-        
+        Calendar date = Calendar.getInstance();
```

Comment:
Potential by-reference bug - I've explained further in another review comment, might need to scroll down a bit!

Reply:
Ah of course - good catch! I'll move them into their respective loops.